mesh/nurbs.hpp

```diff
@@ -61,8 +61,10 @@ class KnotVector
 
    int findKnotSpan(double u) const;
 
-   void CalcShape (Vector &shape, int i, double xi) const;
-   void CalcDShape(Vector &grad,  int i, double xi) const;
+   void CalcShape  (Vector &shape, int i, double xi) const;
+   void CalcDShape (Vector &grad,  int i, double xi) const;
+   void CalcDnShape(Vector &gradn, int n, int i, double xi) const;
+   void CalcD2Shape(Vector &grad2, int i, double xi) const { CalcDnShape(grad2, 2, i, xi);}
```

Comment:
Can you wrap this line since it longer than 80 characters.

Reply:
Yes done.
I think I did perform a "make style" but apparently it did not catch it.
Would it be possible to add this option? Or is it included and I just did not do the "make style"?